skyvern/forge/sdk/api/llm/utils.py

```diff
@@ -29,17 +31,37 @@ async def llm_messages_builder(
                     },
                 }
             )
-
+    # Anthropic models seems to struggle to always output a valid json object so we need to prefill the response to force it:
+    if SettingsManager.get_settings().ENABLE_ANTHROPIC:
```

Comment:
`ENABLE_ANTHROPIC=True` doesn't mean that we're using Anthropic to send the current LLM request. Also, it's possible to use Anthropic through AWS Bedrock. 

I think a good way to move forward here is to update [LLMConfig](https://github.com/Skyvern-AI/skyvern/blob/240551f68684a42cd54a1af6167bfa5eafe5b049/skyvern/forge/sdk/api/llm/models.py#L9) and implement an `add_assistant_prefix` property. 

This way, we can configure assistant prefill manually and always make sure we are checking the language model provider + language model to make an accurate decision.

Required changes:
- [ ] Add `add_assistant_prefix` field to LLMConfig with a default `False` value
- [ ] Update the existing Anthropic `LLMConfig`s to have this field as `True`
    - Can be found [here](https://github.com/Skyvern-AI/skyvern/blob/240551f68684a42cd54a1af6167bfa5eafe5b049/skyvern/forge/sdk/api/llm/config_registry.py#L61-L100)
- [ ] We're going to need to pass the `add_assistant_prefix` to `llm_message_builder` and check that instead of the `ENABLE_ANTHROPIC/ENABLE_BEDROCK` settings.

Reply:
Maybe we can do this in a follow up? It's possible BEDROCK may have a different schema for defining this, similar to how[ OpenAI's JSON mode](https://platform.openai.com/docs/guides/text-generation/json-mode) is different than [Azure Open AI's ](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/json-mode?tabs=python)

OK It's not really that different, but in theory it could be so we can defer this until later!

Reply:
Implemented the mentioned changes in #293